README.md

```diff
@@ -689,7 +689,20 @@ Translations of the guide are available in the following languages:
 <sup>[[link](#rdoc-conventions)]</sup>
 
 * <a name="120-character-limits"></a>
-  Limit lines to 120 characters.
+  Limit lines to 120 characters. When splitting long lines that include
+lists of items, e.g. parameter lists, put each item on a new line.
+
+  ```ruby
+  # bad - hard to visually parse
+  some_method("this is one long parameter", another_long_parameter,
+    a_third_parameter)
+
+  # good
+  some_method(
+   "this is one long parameter",
```

Comment:
I personally prefer the first argument align with the opening paren in these cases.